lua/autorun/client/cl_roundinfo_receive.lua

```diff
@@ -98,19 +99,16 @@ if CLIENT then
 			return
 		end
 
+		local wep_className = net.ReadString()
+		local wep_class = weapons.GetStored(wep_className) or {}
```

Comment:
Does it really make sense to continue (not return) if the weapon is not found?

Reply:
I restructured it. In the case that no stored weapon with that class name exists we will treat it as if the person was killed with no weapon. Please let me know what you think.

Reply:
yea that would be the best solution IMO :)